raiden/tests/integration/contracts/fixtures/contracts.py

```diff
@@ -96,8 +114,8 @@ def token_network_proxy(deploy_client, token_network_contract, contract_manager)
     )
 
 
-@pytest.fixture
-def token_contract(deploy_client, contract_manager):
+@pytest.fixture(name='token_contract')
+def deploy_network_and_return_jsonrpc_proxy(deploy_client, contract_manager):
```

Comment:
s/deploy_network/deploy_token

Reply:
Hummm, weird, it's showing the wrong lines here:

![image](https://user-images.githubusercontent.com/310139/52345818-47d73500-2a05-11e9-865e-3ae1a7b97f5c.png)

Reply:
I think I commented on the first commit by error.